src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
+  .${PLACEHOLDER_CLASS}::before {
+    position: absolute;
+    content: '${text}';
+    cursor: text;
+  };
+</style>`;
+
+    span.className = PLACEHOLDER_CLASS;
```

Comment:
Well we just need something to hook into ourselves. Honestly I'd be keen to make it a super random class so they can't hook into it.